src/main/java/org/isf/menu/gui/Login.java

```diff
@@ -122,6 +124,29 @@ public void keyReleased(KeyEvent event) {
 	private User returnUser;
 	private boolean usersListLogin;
 
+	
+	public Login(JFrame parent) {
+		super(parent, MessageBundle.getMessage("angal.login.title"), true);
+
+		usersListLogin = GeneralData.getGeneralData().getUSERSLISTLOGIN();
+		
+		// add panel to frame
+		LoginPanel panel = new LoginPanel(this);
+		add(panel);
+		pack();
+
+		Toolkit kit = Toolkit.getDefaultToolkit();
+		Dimension screensize = kit.getScreenSize();
+
+		Dimension mySize = getSize();
+
+		setLocation((screensize.width - mySize.width) / 2,
+				(screensize.height - mySize.height) / 2);
+
+		setResizable(false);
+		setVisible(true);
```

Comment:
There has been an effort to remove the user of Toolkit, screensize, setLocation, etc. and replace it with something like:
<pre>
         	pack();
		setLocationRelativeTo(null);
		setResizable(false);
		setVisible(true);
</pre>